schemas/src/main/java/com/scottlogic/deg/schemas/v3/RuleSerializer.java

```diff
@@ -14,24 +14,15 @@ public void serialize(
         SerializerProvider serializerProvider)
         throws IOException {
 
-        if (rule.rule == null) {
-
-            for (ConstraintDTO constraint : rule.constraints)
-            {
-                jsonGenerator.writeObject(constraint);
-            }
-        }
-        else {
-            jsonGenerator.writeStartObject();
-            jsonGenerator.writeFieldName("rule");
-            jsonGenerator.writeString(rule.rule);
-            jsonGenerator.writeArrayFieldStart("constraints");
-            for (ConstraintDTO constraint : rule.constraints)
-            {
-                jsonGenerator.writeObject(constraint);
-            }
-            jsonGenerator.writeEndArray();
-            jsonGenerator.writeEndObject();
+        jsonGenerator.writeStartObject();
```

Comment:
Unit test?

Reply:
See `ProfileSerialiserTests` in the schemas project.

Reply:
Tests updated, additional test created to assert serialisation of rule 'name'